app/controllers.rb

```diff
@@ -65,8 +66,13 @@ def get_search_args_from_params(params)
     distance: params.delete("distance"),
     fields: (params.delete('fields') || "").split(','),
     per_page: params.delete("per_page") || DataMagic.config.page_size,
-    page: params.delete("page").to_i || 0
+    page: params.delete("page").to_i || 0,
+    add_aggregations: params.delete("_command") == 'stats' ? true : false,
```

Comment:
Heh. True... In some cases, Ruby does not evaluate some conditional operators to Booleans (e.g., `1 && 0`), so I tend to get over-cautious.